code/modules/clothing/clothing.dm

```diff
@@ -38,17 +40,8 @@
 	// THESE OVERRIDE THE HIDEHAIR FLAGS
 	var/dynamic_hair_suffix = ""//head > mask for head hair
 	var/dynamic_fhair_suffix = ""//mask > head for facial hair
-	var/greyscale = FALSE

```

Comment:
I was under the assumption that equipping it was enough to make sure the neccesary icon updates worked and gave it the proper coloring, at least from what I understood from Remie's earlier comment.